queuebot/cogs/queue/__init__.py

```diff
@@ -598,6 +603,11 @@ async def test(self, ctx, suggestion: PartialSuggestionConverter = None):
                 suggestion = (None, ctx.message.attachments[0].proxy_url)
             else:
                 raise commands.BadArgument("Couldn't resolve to suggestion or image.")
+        elif not (any(role.id in ctx.bot.maker_roles for role in ctx.author.roles) and any(role.id in ctx.bot.council_roles for role in ctx.author.roles)):
+            await ctx.send(f"{red_tick} It appears that you attempted to test an emoji. Emoji arguments are restricted to Blob Maker+ due to constant misuse of this command.\nIf you want to legitimately test a blob that you have created yourself, then attach your blob's image file to the command instead.")
```

Comment:
We use a max line length of 120.

```suggestion
            await ctx.send(
                f"{red_tick} It appears that you attempted to test an emoji. "
                f"Emoji arguments are restricted to Blob Maker+ due to constant misuse of this command.\n"
                f"If you want to legitimately test a blob that you have created yourself, "
                f"then attach your blob's image file to the command instead."
            )
```